contracts/0.8.25/vaults/OperatorGrid.sol

```diff
@@ -0,0 +1,448 @@
+// SPDX-FileCopyrightText: 2025 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+// See contracts/COMPILERS.md
+pragma solidity 0.8.25;
+
+import {OwnableUpgradeable} from "contracts/openzeppelin/5.2/upgradeable/access/OwnableUpgradeable.sol";
+import {AccessControlEnumerableUpgradeable} from "contracts/openzeppelin/5.2/upgradeable/access/extensions/AccessControlEnumerableUpgradeable.sol";
+
+import {ILidoLocator} from "contracts/common/interfaces/ILidoLocator.sol";
+import {IStakingVault} from "./interfaces/IStakingVault.sol";
+import {VaultHub} from "./VaultHub.sol";
+
+struct TierParams {
+    uint256 shareLimit;
+    uint256 reserveRatioBP;
+    uint256 rebalanceThresholdBP;
+    uint256 treasuryFeeBP;
+}
+
+contract OperatorGrid is AccessControlEnumerableUpgradeable {
+
+    bytes32 public constant REGISTRY_ROLE = keccak256("vaults.OperatorsGrid.Registry");
+
+    /// @notice Lido Locator contract
+    ILidoLocator public immutable LIDO_LOCATOR;
+
+    /// @notice Default group address
+    address public constant DEFAULT_GROUP_ADDRESS = address(1);
+
+    // -----------------------------
+    //            STRUCTS
+    // -----------------------------
+    struct Group {
+        address operator;
```

Comment:
do we need this field?

Reply:
yes, uses as a "group identificator" (checks operator uniques)